experimental/packages/otlp-exporter-base/src/configuration/otlp-http-configuration.ts

```diff
@@ -27,32 +27,34 @@ import type * as https from 'https';
 
 export interface OtlpHttpConfiguration extends OtlpSharedConfiguration {
   url: string;
-  headers: Record<string, string>;
+  headers: () => Record<string, string>;
```

Comment:
Hi @pichlermarc, I have done a similar pr about that for string or function support https://github.com/open-telemetry/opentelemetry-js/pull/5118 
What do you think??

Reply:
Hi @pichlermarc, do you have any idea when the team will release the version with this commit? thanks